Patch-Source: https://github.com/serge-sans-paille/pythran/pull/2104
needed for scipy which is cxx 14
--
From 7e9c31405f995196074add225864d52773e5321e Mon Sep 17 00:00:00 2001
From: serge-sans-paille <serge.guelton@telecom-bretagne.eu>
Date: Tue, 2 May 2023 14:24:34 +0200
Subject: [PATCH] Backport "Defer to (good) compiler the task of computing
 ln10" commit from xsimd

Commit a1e07510dd81a0b074d2cef8e09bf8e59b926aae. It's a compatibility
patch for C++11.
---
 third_party/xsimd/arch/xsimd_scalar.hpp | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/third_party/xsimd/arch/xsimd_scalar.hpp b/third_party/xsimd/arch/xsimd_scalar.hpp
index 41a44709d..fc227db84 100644
--- a/third_party/xsimd/arch/xsimd_scalar.hpp
+++ b/third_party/xsimd/arch/xsimd_scalar.hpp
@@ -497,11 +497,13 @@ namespace xsimd
 #else
     inline float exp10(const float& x) noexcept
     {
-        return std::exp(0x1.26bb1cp+1f * x);
+        const float ln10 = std::log(10.f);
+        return std::exp(ln10 * x);
     }
     inline double exp10(const double& x) noexcept
     {
-        return std::exp(0x1.26bb1bbb55516p+1 * x);
+        const double ln10 = std::log(10.);
+        return std::exp(ln10 * x);
     }
 #endif
 
